### PR TITLE
Feature/jenkins 36211

### PR DESCRIPTION
**Pull request by scherler**
Related to issue # . 
36211

Summary of this pull request: 
- [x] Implement pipeline channel sse events
- [x] pick up sse events in detail view to refetch when needed
- [ ] stop follow when scrolling away -> should it be part of the PR?

[Karaoke](https://youtu.be/oHbVg5Tm-Q8)

@reviewbybees

### CODE REVIEW

**Comment by scherler** [comment]
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11:09 min
[INFO] Finished at: 2016-06-28T20:20:41+00:00
[INFO] Final Memory: 84M/640M

last build, weird

**Comment by michaelneale** [comment]
Wow awesome. I was laughing so hard when the music started on that video. 😂

By "scrolling away" you mean taking control from user? I guess that could be another PR. 
cc @i386 

As for build failure, I think that is an intermittent test we habve seen before (unrelated) - retrying.

**Comment by i386** [comment]
@michaelneale @scherler another PR but not another ticket.

**Comment by michaelneale** [comment]
OUTSTANDING!



This needs updating to master (sorry) as @kzantow's js-extensions extensionpoint refactor went in, but probably mostly mechanical (sorry!). 

But this looks really good to me. I tried it with non multibranch and it worked great (for multibranch, Tom has a PR open to ensure events are delivered so that it picks up the new runs etc). 

LGTM

**Comment by scherler** [comment]
:) yeah we are starting to rock real hard , ;) ,,,anyway so :bee: ? 
margarita

**Comment by michaelneale** [comment]
:bee:

**Comment by scherler** [comment]
rofl chris high five

deadpooltaxi

**Comment by scherler** [comment]
@reviewbybees done

File: blueocean-dashboard/src/main/js/redux/actions.js [hunk 1]
```diff
@@ -712,7 +652,7 @@ export const actions = {
                 let nodeModel;
                 let node;
                 if (!config.node) {
-                    const focused = information.model.filter((item) => item.isFocused)[0];
+                    const focused = information.model.filter((item) => item.isFocusedq)[0];
```
Comment (tfennelly) [inline blueocean-dashboard/src/main/js/redux/actions.js]: `isFocusedq` ? `q` looks like a typo.

**Comment by tfennelly** [comment]
LGTM 🐝 

A few tests for the `UrlUtils` would be great too.

**Comment by scherler** [comment]
@tfennelly will wait for jenkins and merge afterwards. All calculate* functioned are covered by tests.

**Comment by tfennelly** [comment]
@scherler you are a legend my friend !!

**Pull request merged**